apiary.apib

```diff
@@ -3313,6 +3319,18 @@ Create a new milestone.
         + due_on: `2018-01-01` (string, required)
         + name: `Initial setup` (string, required)
         + responsible_user_id: `e1240972-6cfc-4549-b49c-edda7568cc48` (string, required)
+        + One Of
+            + Properties
+                + `billing_method`: `time_and_materials` (enum) - If omitted, `time_and_materials` is used
+                    + Members
+                        + `non_invoiceable`
+                        + `time_and_materials`
+                + budget (Money) - If omitted, the budget will default to `0.00`
```

Comment:
Is this because of legacy reasons? Could we find a way to keep the budget nullable and return `null` instead of `0.00`?

Reply:
That's quite a big change I'm afraid. But it is most definitely on our list. So yeah, it's because of legacy

Reply:
Pitty :(